devices/chipdb_xilinx.cmake

```diff
@@ -103,6 +105,7 @@ function(generate_xc7_device_db)
     set(device ${create_rapidwright_device_db_device})
     set(part ${create_rapidwright_device_db_part})
     set(device_target ${create_rapidwright_device_db_device_target})
+    set(family ${create_rapidwright_device_db_family})
```

Comment:
I have noticed an inconsistency which went un-noticed. The `cmake_parse_arguments` variable name is `create_rapidwright_device_db` which is not correct and it should be `generate_xc7_device_db`. All the above variables should use the `generate_xc7_device_db` prefix as well

Reply:
Fixed